components/elm/src/main/pftvarcon.F90

```diff
@@ -303,6 +311,21 @@ module pftvarcon
   real(r8), allocatable :: gcbr_p(:)           !effectiveness of roots in reducing rainfall-driven erosion
   real(r8), allocatable :: gcbr_q(:)           !effectiveness of roots in reducing runoff-driven erosion
 
+
+  !---------------------- user-defined parameter file ---------------------------------------------------------------------
+  ! NOTE: using 'mergetoelmpft' variable in nc parameter file as a flag
+  integer :: nndllf0_tree, nndllf_tree          ! 0-based pft index for first, last type of needle-leaf tree
```

Comment:
See if we can get rid of all the start/end block numbers and just specify with flags